cflib/positioning/position_hl_commander.py

```diff
@@ -141,7 +145,7 @@ def __enter__(self):
         return self
 
     def __exit__(self, exc_type, exc_val, exc_tb):
-        self.land()
+        self.land(landing_height=self._default_landing_height)
```

Comment:
With the proposed change to the `land()` method, `__exit__()` should only call `land()` without any arguments